src/components/m-col/index.js

```diff
@@ -35,7 +35,7 @@ const modifierProp = (filter, format) => (props, propName, componentName) => {
   const modifiers = toArray(prop);
 
   if (prop && modifiers.length !== modifiers.filter(validModifiers).length) {
-    console.log(props, modifiers);
+    // console.log(props, modifiers);
```

Comment:
upps, that log can be deleted, may I ask you to do so?

Reply:
done.